client/web/src/App.vue

```diff
@@ -196,10 +196,12 @@ img {
 
 <script lang="ts">
 import { defineComponent } from "vue";
+import fullscreen from "@/utilities/fullscreen";
 import MainWindow from "./components/window/MainWindow.vue";
```

Comment:
Probably avoid mixing relative with absolute paths.

Reply:
Will be fixed in an upcoming PR which replaces all paths in the project with absolute ones.